modules/ROOT/pages/whatsnew.adoc

```diff
@@ -28,6 +28,9 @@ This can only happen during a mixed mode of SGW with a 3.2 version and an older
 * *Eventing Support with Sync Gateway 3.2.0+ and Couchbase Server 7.6.3+*
 
 Sync Gateway 3.2.0 now supports improved interoperability with Eventing from Couchbase Server version 7.6.3+.
+
+
+You can now create Eventing functions with read write bindings with the source bucket associated with a Sync Gateway database.
```

Comment:
"read/write bindings" better?

Reply:
Had a quick gander in dictionaries etc, seems to be hyphenated! Changed